src/rules.ts

```
@@ -0,0 +1,42 @@
+let rules = [
+  // 0. expand triangles
+  "00000100010110010100010001110000101100001100101001001100100000100001000011001010000000000000000000100001000100000000000100000010100000110000000000000000000101000000001000000000100000110000000000000000001000101000000110011000000000000000101000010000000101000000000000000000000001000000011011000000000000000000000101100010010000001000000000000000000010000000010000000001000010100101010001100000001000011001111000110100000000000000000110110000100100000001000000110000000000000100001000100000000000110000001010110001",
+  // 1. dots
+  "01101101111111101111101111111111010111101011111110111110111111111110110110010111100111111111101101111101111101101111111110111001000111111011001110011011100101111101100111010111111111111101111111010111111110111101101110101101110101111101101111110010111111111011111111111010100111111010111111011100111111011111100111101111111100111111011011010101111011111001111011111010000111110111011111011101011111001011011110011111111111111011010101110111101011111101111111111111111111110111111111111111111111101110011111111111",
+  // 2. tides
+  "10011101111110011001110111111111011011101111011111110100111101111111111111111111111111111111101111000001111111111110110101011111111111110110011111110011011011111110111111111110111110101110111111111111111101111101110110101011110011111110011111111111101111101111101111110111001111111111101111111110111101101101110101111111111111111110111111111011111011110110111011110101101010101111110110101111110111110111111100110111010111101001101111111111111111111111100111110011111110111111110011011111101111111110101111111111",
+  // 3. beach
+  "11111111101111011111110111011111111011010110110111111111111111011111001111111001010111111111110111100110111011111100111101111011110111111111111111111100111101111111100110111111111011111101101111110111010010111111101111101111111111110111001111111111110001011111111111110110101111111111111111110111111111011111101100111111101111100011101101111111111100111111111111111111111111101111111111111011110111110011111101111111111111111110111111111011101111001111111110101110111111011100101101111100101011101111111101110111",
+].map((x) => x.split("").map(Number));
+
+export let getRules = () => {
```

Biome

[error] This let declares a variable that is only assigned once.

'getRules' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)